reducer/src/test/java/com/graphicsfuzz/reducer/util/SimplifyTest.java

```diff
@@ -122,4 +122,24 @@ public void testMacroBlockRemoved() throws Exception {
     CompareAsts.assertEqualAsts(expected, simplifiedTu);
   }
 
+  // TODO(491): Enable once issue 491 is fixed.
+  @Ignore
+  @Test
+  public void testIdentityNotNestedRemoved() throws Exception {
+    final TranslationUnit tu = ParseHelper.parse("void main() {"
+        + " if(_GLF_IDENTITY(1, 1)) {}"
```

Comment:
I would prefer this to be _GLF_IDENTITY(true, true), since if(1) is not legal GLSL.

Reply:
Done